Procfile.dev

```diff
@@ -1,4 +1,4 @@
 web: env PORT=3000 RAILS_ENV=development bundle exec puma -C config/puma.rb
 sidekiq: env PORT=3000 RAILS_ENV=development bundle exec sidekiq
-stream: env PORT=4000 yarn run start
+stream: env PORT=4000 cd streaming && yarn start
```

Comment:
If this is in the workspace, I think it can be something like `yarn workspace streaming start` ?

Reply:
oooh, good call!

```suggestion
stream: env PORT=4000 yarn workspace @mastodon/streaming start
```